Penumbra/Mods/ModManager.cs

```diff
@@ -102,25 +103,14 @@ public void DiscoverMods( DirectoryInfo basePath )
             // Needed to reload body textures with mods
             //_plugin.GameUtils.ReloadPlayerResources();
         }
-        private (InstallerInfo, Option, string) GlobalPosition( string rel, Dictionary<string, InstallerInfo> gps )
-        {
-            string filePath = null;
-            foreach( var g in gps )
-            {
-                foreach( var opt in g.Value.Options )
-                {
-                    if( opt.OptionFiles.TryGetValue( rel, out filePath ) )
-                    {
-                        return (g.Value, opt, filePath);
-                    }
-                }
-            }
-            return (default( InstallerInfo ), default( Option ), null);
-        }
+
         public void CalculateEffectiveFileList()
         {
             ResolvedFiles.Clear();
             SwappedFiles.Clear();
+
+            if (!_plugin.Configuration.IsEnabled)

```

Comment:
should do this in the hook instead of disabling conflict notifications and etc as this is useful outside of just disabling things entirely, this would also have no effect if you disable it once something has loaded